packages/sources/svelte/src/components/navigation/VtmnSearch/enums.js

```diff
@@ -0,0 +1,11 @@
+export const VARIANTS = {
+  DEFAULT: 'default',
+  GHOST: 'ghost',
```

Comment:
No ghost variant for the search
```suggestion
  
```

Reply:
+1 w/ @GaspardMathon, there's no "ghost" variant.
Not in our Storybook CSS, neither in Figma: https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/Vitamin---Web-UI-kit-(v0)?node-id=6527%3A15021